psi4/driver/driver.py

```diff
@@ -872,6 +872,70 @@ def properties(*args, **kwargs):
     else:
         return core.variable('CURRENT ENERGY')
 
+def optimize_geometric(name, **kwargs):
+
+    # will move imports to top of file when i'm sure where this method goes
+    # TODO: check if GeomeTRIC is installed at the start
```

Comment:
can do something like this to check geometric's presence https://github.com/MolSSI/QCElemental/blob/master/qcelemental/molutil/align.py#L416-L419

Reply:
actually, nevermind this. once qcng is imported, just check if geometric is `.found(raise_error=True)` and that'll handle the warning, I think.

Reply:
very useful to know. geometric is now explicitly imported, so i guess we can just let psi throw an import error now